scripts/travis/test.sh

```diff
@@ -14,6 +14,8 @@ curl -sL -o ~/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gim
 chmod +x ~/gimme
 eval "$(~/gimme "${GOLANG_VERSION}")"
 
+"${SCRIPTPATH}/../buildtools/install_buildtools.sh"
```

Comment:
installing too many "tools" was one of the reason travis build was slow. 
Add an optional argument to `install_buildtools.sh` so that only gotestsum would be installed.